src/cfml/system/modules_app/server-commands/commands/server/prune.cfc

```diff
@@ -0,0 +1,159 @@
+
+                for( currentServer in servers ){
+                    result = dateDiff( "m", servers[ currentServer ].dateLastStarted, now() );
+                    if ( result >= arguments.months ){
+                        arrayAppend( filterServers, servers[ currentServer ] );
+                    }
+                }            
+
+                if ( arguments.force ){
+
+                    print.line( "using prune with force...", 'yellow' ).toConsole();
+
+                    var runningServers = getRunningServers( servers );
+                    /* areThereRunningServers = ( ! runningServers.isEmpty() ) */
+                    /* print.line( "are there running servers: #areThereRunningServers#" ).toConsole(); */
+
+                    if ( !runningServers.isEmpty() ) {
+
+                        var stopMessage = "Stopping server #serverInfo.name# first....";
+        
+                        print.line( stopMessage )
+                            .toConsole();
+        
+                        runningServers.each( function( ID ){
+                            var stopResults = serverService.stop( runningServers[ arguments.ID ] );
+                            print.line( stopResults.messages, ( stopResults.error ? 'red' : 'green' ) )
+                                .toConsole();
+                        } );
+        
+                        // Give them a second or three to die or file locks will still be in place (on Windows, at least)
+                        // This is hacky, but there's no clean way to poll for when the process is 100% dead
+                        sleep( 3000 );
+                    }
+
+                    for( currentServer in filterServers ){
+
+                        print.line( serverService.forget( #currentServer.name# ), 'red' )
+                            .toConsole();
+                    }
+
+                } else {
+                    
+                    for( currentServer in filterServers ){
+
+                        var askMessage = "Really forget & delete server '#currentServer.name#' forever [y/n]?";
+            
+                        if( confirm( askMessage ) ){
+                            print.line( serverService.forget( #currentServer.name# ), 'red' )
+                                .toConsole();
+                        } else {
+                            print.line( "Cancelling forget '#currentServer.name#' command", 'blue' )
+                                .toConsole();
+                        }        
+                
+                    }     
+
+                
+                }
+
+            }
+
+        }
+
+
+    }
+
+    private function generatePruneListServers( required string months, required struct servers ){
+
+        /* user wants to see the list of servers with given months */
+        print.line( "generating list servers to be pruned...", 'yellow' ).toConsole();
+        serverCount=0;
```

Comment:
This separate counter variable is unnecessary as the size of the `serverData` tells you how many servers matched.